cou/apps/app.py

```diff
@@ -67,6 +72,9 @@ class Application:
     os_origin: str = ""
     channel: str = ""
     units: defaultdict[str, dict] = field(default_factory=lambda: defaultdict(dict))
+    current_os_release: str = ""
+    next_os_release: str = ""
```

Comment:
isn't `next_os_release` a bit unnecessary? You could return an `OpenStackRelease` object in `current_os_release`, and the caller could use that to determine the next release

Reply:
Good idea. Now the `OpenStackCodenameLookup` returns a list of `OpenStackRelease` and we can access the next release by `next_release` property